src/setupTests.js

```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
+// allows you to do things like:
+// expect(element).toHaveTextContent(/react/i)
+// learn more: https://github.com/testing-library/jest-dom
+import '@testing-library/jest-dom';
```

Comment:
- [ ] As I mentioned above I would like to recommend you remove all the unnecessary files like this one from your project to make it clean. Please remove them :+1: